src/runtime-rs/Makefile

```diff
@@ -88,12 +88,19 @@ HYPERVISOR_FC = firecracker
 HYPERVISOR_QEMU = qemu
 HYPERVISOR_CLH = cloud-hypervisor
 
+##VAR DRAGONBALL=yes|no Define whether dragonball is compiled into runtime-rs.
```

Comment:
```suggestion
## VAR DRAGONBALL=yes | no, which indicates dragonball builtin runtime-rs or not, default yes.
```

Reply:
In other parts of the Makefile, no space is added between ## and VAR. I think there might not be a need to add a space here either, to align with the overall style.

Reply:
useless comments should be removed.